README.md

```diff
@@ -79,10 +79,13 @@ In order to use the code in this demo you will need access to the following tool
 [Kubernetes Engine v1.10.0 or later](https://cloud.google.com/kubernetes-engine/docs/quickstart#before-you-begin) service enabled
 * If you do not have a Google Cloud Platform account you can sign up [here](https://cloud.google.com) and get 300 dollars of free credit on your new account.
 * [Google Cloud SDK (200.0.0 or later)](https://cloud.google.com/sdk/downloads)
-* [ApacheBench](https://httpd.apache.org/docs/2.4/programs/ab.html)
+* [ApacheBench](https://httpd.apache.org/docs/2.4/programs/ab.html) (`sudo apt-get install apache2-utils`)
 * [HashiCorp Terraform (>= v0.11.7)](https://www.terraform.io/downloads.html)
 * [gcloud](https://cloud.google.com/sdk/gcloud/)
 * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* [shellcheck](https://www.shellcheck.net/) (`sudo apt-get install shellcheck`)
```

Comment:
shellcheck, flake8, and hadolint are required only for Development.  We can mention them, but we also have the need for a DEVELOPER.md doc as well.  It has been backlogged.

Reply:
That's cool, but then shouldn't the README be updated with instructions to not use `make` to build the container and deploy the cluster? For example, https://github.com/GoogleCloudPlatform/gke-monitoring-tutorial has instructions for the demo user to run `terraform` commands directly instead of make.

Reply:
You need to use specific make targets to deploy the demo. We want all of our demos to use make, but some of them are further along than others.

Maybe we have the default make target the create? Currently the default target is used for ci.

Reply:
Ah, this is helpful. It definitely seems like there's a tension between raw terraform commands and make files, so wasn't sure if it was a transition in progress or purposeful parallel paths (one for demo user and one for dev/CI). Will update the README then to use make, with note to show user how to see what terraform commands are being run.

Reply:
@milesmatthias no tension the READMEs got out of synch with changes to the code.